package.json

```
@@ -19,6 +19,7 @@
     "lint:fix:ts": "npm run lint:ts -- --fix",
     "lint:fix:json": "prettier -w src/assets/*/*.json",
     "verify": "ts-node scripts/verify.ts",
+    "verifyChanges": "ts-node scripts/verifyChange.ts",
```

These two are confusing, can we rename the first one so we have clearer "what each script verifies". As a suggestion:

```suggestion
    "review:verify-deployment": "ts-node scripts/review/verifyDeployment.ts",
    "review:diff": "ts-node scripts/review/diff.ts",
```

---

(the term verify means more "verify on-chain" to me).

---

Done 👍🏾